modules/ztp-acm-adding-images-to-mirror-registry.adoc

```diff
@@ -0,0 +1,72 @@
+// CNF-1500 ZTP - preparing to install ACM
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+[id="ztp-acm-adding-images-to-mirror-registry_{context}"]
+= Adding {op-system} ISO and RootFS images to a disconnected mirror host
+
+Before you install a cluster on infrastructure that you provision, you must create {op-system-first} machines for it to use. Use a disconnected mirror to host the {op-system} images you require to provision your distributed unit (DU) bare-metal hosts.
+
+.Prerequisites
+
+* Deploy and configure a HTTP server to host the {op-system} image resources on the network. You must be able to access the HTTP server from your computer, and from the machines that you create.
+
+[IMPORTANT]
+====
+The RHCOS images might not change with every release of {product-title}. You must download images with the highest version that is less than or equal to the {product-title} version that you install. Use the image versions that match your {product-title} version if they are available. You require ISO and RootFS images to install {op-system} on the DU hosts. RHCOS qcow2 images are not supported for this installation type.
+====
+
+.Procedure
+
+. Log in to the mirror host.
+. Obtain the {op-system} ISO and RootFS images from link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/[mirror.openshift.com], for example:
+
+.. Export the required image names and {product-title} version as environment variables:
++
+[source,terminal]
+----
+$ export ISO_IMAGE_NAME=<iso_image_name> <1>
+----
++
+[source,terminal]
+----
+$ export ROOTFS_IMAGE_NAME=<rootfs_image_name> <2>
+----
++
+[source,terminal]
+----
+$ export OCP_VERSION=<ocp_version> <3>
+----
+<1> ISO image name, for example, `rhcos-4.8.0-fc.9-x86_64-live.x86_64.iso`
```

Comment:
This is the only time I've seen annotations clumped like this. I don't think it's explicitly forbidden, but it is unique. (I would probably annotate them each separately.) It might be worth checking the portal when this content lands there to make sure that it's rendering well enough there, too.

Reply:
I played around with it. thought it looked better like this..

![image](https://user-images.githubusercontent.com/74046732/125957342-3dafc915-afe7-462a-9fc5-ec7dbbdc5e53.png)